lms/djangoapps/courseware/grades.py

```diff
@@ -33,6 +33,20 @@
 log = logging.getLogger("edx.courseware")
 
 
+def _get_score(*args):
```

Comment:
Not sure I understand why the fix is in `edx-platform` versus in `fun-apps`. If `edx-platform` now expects a dict you may break other things by returning a tuple? 
Should the fix be done in fun-apps? Or  it is a real bug in ora2 then the PR must be submitted to edx/edx-platform rather than on our fork? 
If the problem is with the format of the data stored in the existing database, then the fix should involve a data migration?